src/main/java/software/amazon/smithy/lsp/ServerOptions.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.lsp;
+
+//import javax.print.attribute.standard.Severity;
+import com.google.gson.JsonObject;
+import java.util.Arrays;
+import java.util.Optional;
+import software.amazon.smithy.model.validation.Severity;
+
+public final class ServerOptions {
+    private Severity minimumSeverity = Severity.WARNING;
+    private boolean onlyReloadOnSave = false;
```

Comment:
Suggestion: You can make these final, since at least right now we aren't going to change them after initialization (supporting that would require adding some synchronization anyway). This also lets `Builder` be the source-of-truth for the default values of each option.
```suggestion
    private final Severity minimumSeverity;
    private final boolean onlyReloadOnSave;
```